.circleci/config.yml

```diff
@@ -53,7 +53,15 @@ jobs:
     steps:
       - checkout
       - *restore_venv_cache
-      - *build
+      - &attach_generated_sql
+        attach_workspace:
+          at: /tmp/workspace
+      - &copy_generated_sql
+        run:
+          name: Move generated-sql into place
+          command: |
+            rm -rf sql/
+            cp -r /tmp/workspace/generated-sql/sql sql
```

Comment:
A bunch of CI jobs now depend on the generated-sql. Creating aliases for these steps here to make them reusable